### PR TITLE
Key 'supported_architectures' should be an array and not only a string

### PR DESCRIPTION
**Pull request by jby**
To work properly with munkiadmin according to @hjuutilainen:

https://macadmins.slack.com/archives/C0D7FU2Q1/p1647623269525709
https://macadmins.slack.com/archives/C0D7FU2Q1/p1647632972333489

Plus a cleanup of the name variable according to @gregneagle Munki will handle the different architectures

### CODE REVIEW

File: Docker/Docker.munki.recipe.yaml [hunk 1]
```diff
@@ -11,10 +11,11 @@ Input:
       - testing
     description: Spice up your Dock a little.
     developer: Docker Inc
-    display_name: Docker
-    name: '%NAME%-%ARCHITECTURE%'
+    display_name: '%NAME'
```
Comment (smithjw) [inline Docker/Docker.munki.recipe.yaml:14]: Shouldn't this still be `'%NAME'-%ARCHITECTURE%` to match with the pkg name?
```suggestion
    display_name: '%NAME-%ARCHITECTURE%'
```

File: Docker/Docker.munki.recipe.yaml [hunk 2]
```diff
@@ -11,10 +11,11 @@ Input:
...
+    name: '%NAME%'
     unattended_install: true
-    supported_architectures: '%ARCHITECTURE%'
+    supported_architectures:
+    - '%ARCHITECTURE%'
```
Comment (smithjw) [inline Docker/Docker.munki.recipe.yaml:18]: This line should be indented by two spaces for the proper YAML formatting
```suggestion
      - '%ARCHITECTURE%'
```

**Review by smithjw — CHANGES_REQUESTED** [review]

File: Docker/Docker.munki.recipe.yaml (hunk 2 above)
Comment (jby) [reply Docker/Docker.munki.recipe.yaml]: Done and pushed update - got fooled by my editor

**Comment by eithor** [comment]
Please note that if you change the pkg recipe to just NAME for the pkg-name it will fail for those that use e.g. Jamf and not Munki and require unique pkg names

**Comment by eithor** [comment]
> > Please note that if you change the pkg recipe to just NAME for the pkg-name it will fail for those that use e.g. Jamf and not Munki and require unique pkg names
> 
> Sure, but this PR was directed specifically towards a Munki recipe...

@jby 
I understand, though I was thinking about your comment above (second from the top) where you say that the pkg-recipe also needs to be changed.

**Comment by smithjw** [comment]
@jby @eithor I'm probably going to close this PR as I've created a new set of Universal recipes that pkg both Intel and Arm versions into the one pkg. Yes, it's significantly larger because it's packaging both installers, but also a lot simpler for admins.

**Pull request closed without merging**